qemu/tests/cfg/nmi_bsod_catch.cfg

```diff
@@ -2,6 +2,8 @@
     type = nmi_bsod_catch
     only Windows
     mem_fixed = 2048
+    Win2016, Win2019:
+        mem_fixed = 4096
     enable_pvpanic = no
```

Comment:
why we use "mem_fixed"? In case the dump is too large? And why ws2016/2019 is 4G, others are 2G?

Reply:
yes, mem_fixed = 2048  is original setting, I think it's want to avoid the dump is too large.
For ws2016 and ws2019, the guest vm_mem_min = 4096,  if remain keep  mem_fixed = 2048, the mem will less than vm_mem_min, the case will be cancled, so change it to 4096.